amberdata/test/balance.test.ts

```diff
@@ -1,7 +1,7 @@
-import { makeExecute } from '../src/adapter'
+import adapter from '../src/index'
```

Comment:
Could destructure this
```suggestion
import { makeExecute }  from '../src/index'
```
Just a style preference thing.

Reply:
There are some issues on the `export =` ES6 default exports of adapters' index.ts files, that's why it can't be used as the other imports. But it should be converted to this syntax once we resolve this.